CITATION.cff

```diff
@@ -0,0 +1,41 @@
+cff-version: 1.2.0
+message: "Please cite this software using these metadata."
+type: software
+title: Shapely
+version: "2.0"
+# date-released: "2022-MM-DD"
+# doi: 10.5281/zenodo.#######
+abstract: "Manipulation and analysis of geometric objects in the Cartesian plane."
+repository-artifact: https://pypi.org/project/Shapely
+repository-code: https://github.com/shapely/shapely
+license: "BSD-3-Clause"
+authors:
+  - given-names: Sean
+    family-names: Gillies
+    orcid: https://orcid.org/0000-0002-8401-9184
+  - given-names: Casper
+    family-names: "van der Wel"
+    affiliation: "Nelen & Schuurmans"
+    orcid: https://orcid.org/0000-0002-0488-2237
+  - given-names: Joris
+    family-names: "Van den Bossche"
+    orcid: https://orcid.org/0000-0003-3284-2977
+  - given-names: "Mike W."
+    family-names: Taves
+    email: mwtoews@gmail.com
+    affiliation: "GNS Science"
+    orcid: https://orcid.org/0000-0003-3657-7963
+  - family-names: Arnott
```

Comment:
Minor nit: for easier maintenance, it would be good if given / family names were in consistent order